homebrew/gerbil-scheme.rb

```diff
@@ -0,0 +1,35 @@
+class GerbilScheme < Formula
+# This .rb file is tangled (AKA generated) from README.org
+  desc "Opinionated dialect of Scheme designed for Systems Programming"
+  homepage "https://cons.io"
+  license any_of: ["LGPL-2.1-or-later", "Apache-2.0"]
+  url "https://github.com/drewc/gerbil.git", using: :git, branch: "homebrew-diff"
+  head "file:///Users/drewc/me/src/gerbil-diff", using: :git, branch: "homebrew-diff"
+  version "17.9"
```

Comment:
and this should `v0.18-rc1`

Reply:
for the "version" field, I'm going to put version: 0.18 revision:0 as, in my head, 'v0.18-rc1` is greater than "0.18.0" so I suspect homebrew is the same and we want "revisions" to work. The revisions do not need to match our point releases, they are just there to tell homebrew this has already been installed and/or needs updating. I also won't change the revision for release, so 0.18.0 will be pushed to homebrew upstream upon release and anyone beta-testing that formula will deal with the "tell homebrew to re-install" themselves.